.tool-versions

```diff
@@ -1,2 +1,2 @@
-elixir 1.7.4-otp-20
-erlang 20.0
+elixir 1.8.2-otp-21
```

Comment:
I ended up figuring out my problem regarding the `-otp-20` need. I think I installed elixir 1.7.4 before asdf started compiling it with opt 19 be default. Just needed to reinstall and then it was fixed.

We can remove the otp suffix if you want, but it doesn't really hurt anything by being there.

Reply:
I kept the suffix so that we run the same version in dev as on Travis: [link](https://travis-ci.org/keathley/wallaby/jobs/563091366#L456)
Looks like asdf compiles 1.8.2 with otp 20 by default:
```
$ asdf install
elixir 1.8.2 is already installed
erlang 21.3 is already installed
$ elixir -v
Erlang/OTP 21 [erts-10.3] [source] [64-bit] [smp:6:6] [ds:6:6:10] [async-threads:1] [hipe]

Elixir 1.8.2 (compiled with Erlang/OTP 20)
```